### ISSUE TITLE
html.elements.input.input-time - is now supported by safari

### ISSUE THREAD

**Issue by timstokman** [mdn/browser-compat-data#9172]
#### What information was incorrect, unhelpful, or incomplete?

The page claims no support with safari for this input type.

#### What did you expect to see?

Safari added support for this in october 2020, see: https://bugs.webkit.org/show_bug.cgi?id=214946 and https://bugs.webkit.org/show_bug.cgi?id=200416

It's supported from TP 115 onwards apparently.

#### Did you test this? If so, how?

I asked a colleague to test this jsfiddle on his iPad: https://jsfiddle.net/2etfd6vm/

<details>
<summary>MDN page report details</summary>

* Query: `html.elements.input.input-time`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time
* Report started: 2021-02-18T14:31:31.818Z

</details>

**Comment by ddbeck** [comment]
Thanks for reporting this, @timstokman.

Related to this is @tari's PR https://github.com/mdn/browser-compat-data/pull/9351, which attempts to make this change (and a couple of others) for desktop Safari.

For several reasons, we don't track features in Safari TP (at least not yet). When there's a general release Safari version number to pin this to, we can update the data. Until then, this will have to wait.

**Comment by timstokman** [comment]
Works fine on the iPad without using a TP version of safari, so I'm assuming it made its way into the regular release. Here: https://webkit.org/status/#feature-date-and-time-input-types it says that only the week and month input types are not fully supported, time is.

**Comment by tari** [comment]
My understanding is that mobile Safari has supported these input types for a long time; the recent addition is for desktop Safari. The data already say support exists on iOS.

**Comment by ddbeck** [comment]
Ah, it looks like I misunderstood the desktop/iOS distinction. Looking this over again, here's a summary of the situation:

| feature | iOS support | macOS support |
| --- | --- | --- |
| [`date`](https://github.com/mdn/browser-compat-data/blob/master/html/elements/input/date.json) | 5 | `false`, noted as "The input type is recognized, but there is no date-specific control." (shipping in a future version) |
| [`time`](https://developer.mozilla.org/docs/Web/HTML/Element/input/time) | `true` (missing version) | `false` (shipping in a future version) |
| [`datetime-local`](https://github.com/mdn/browser-compat-data/blob/master/html/elements/input/datetime-local.json) | `true` (missing version) | `false`, noted as "The input type is recognized, but there is no date-specific control." (shipping in a future version) |
| [`month`](https://github.com/mdn/browser-compat-data/blob/master/html/elements/input/month.json) | `true` (missing version) | `false` |
| [`week`](https://github.com/mdn/browser-compat-data/blob/master/html/elements/input/week.json) | `true` (according to WebKit platform status, should be `false`) | `false` |

It looks like the only obvious change we could make now is to correct `week` to false for both platforms. I've opened https://github.com/mdn/browser-compat-data/pull/9393 to do that.

We could also fill in missing version numbers for Safari for iOS. If you know of version numbers for that data or can test for them, I would welcome a PR amending that data.

I'm hoping the changes on desktop either appear in a release soon (or we merge some of the implementation pending schema changes that have been proposed). Until then, I think the main issue here and PR #9351 are on hold.

**Comment by jakub-g** [comment]
https://twitter.com/jensimmons/status/1387445020555419649

```
Safari 14.1 (shipping this week) also added support for Date & Time form controls:
<input type="datetime-local">
<input type="date">
<input type="time">
on macOS.
```

**Comment by vinyldarkscratch** [comment]
This has been resolved by #10129!
